airbyte-ci/connectors/metadata_service/orchestrator/orchestrator/assets/registry.py

```diff
@@ -92,12 +132,12 @@ def metadata_to_registry_entry(metadata_entry: LatestMetadataEntry, connector_ty
     overrode_metadata_data["public"] = True
 
     # if there is no releaseStage, set it to "alpha"
-    # Note: this is something our current cloud registry generator does
-    # Note: We will not once this is live
     if not overrode_metadata_data.get("releaseStage"):
         overrode_metadata_data["releaseStage"] = "alpha"
 
+    # apply generated fields
     overrode_metadata_data["iconUrl"] = metadata_entry.icon_url
+    overrode_metadata_data["releases"] = apply_connector_release_defaults(overrode_metadata_data)
```

Comment:
In the case of `if metadata_releases is None: return None`, is this setting to `releases: null` instead of leaving the key out entirely? If so is that the behavior we want? (I don't think it matters on the consumption side, but might look a little confusing)

Reply:
Good catch. It will set it to null. Which I think is a fine trade off to being able to keep the "if" statement isolated to `apply_connector_release_defaults`